content/docs/user-guide/assets/pipeline/asset-builders.md

```diff
@@ -1,71 +1,190 @@
+| debug_create       | Debug mode for the create job of the specified file.                                                                                                                                                          |
+| debug_process      | Debug mode for the process job of the specified file.                                                                                                                                                         |
+| tags               | Additional tags to add to the debug platform for job processing. One tag can be supplied per option.                                                                                                          |
+| platform           | Platform to use for debugging, for example, `pc` |
+
+## Loading other files
+
+Sometimes, when authoring a builder, it may be necessary to load other files besides the primary source file to finish a given process. Here are some scenarios that this can come up, and best practices for handling them.
+
+### Understanding file locations
+
+![File Locations](/images/user-guide/assets/pipeline/asset_builders/asset_builder_file_locations.png)
+
+Files relevant to asset processing can exist in these locations:
+1. The Asset Cache is where all product assets exist.
+1. Scan directories are the only place source assets can exist.
+1. Source files (not assets) can be on any drive in any location.
+
+Scan directories are mostly these locations. Scan directories can be elsewhere, but these are the primary ones. Read more about [scan directories here:](/docs/user-guide/assets/pipeline/scan-directories/)
+1. The current project's directory.
+1. The `Assets` folder for each Gem enabled for the project.
+
+Individual Gems, the O3DE project in use, and the engine itself can all be installed to different drives.
+
+#### Types of files in asset processing
+
+![All files in asset processing](/images/user-guide/assets/pipeline/asset_builders/asset_builder_all_dependencies.png)
+
+* Source assets
+   * Source assets are files that exist in scan directories for an O3DE project, that match an Asset Builder's registered pattern, and result in a job being created when create jobs is called with the source asset for the Asset Builder.
+   * Read more about [source assets here.](/docs/user-guide/assets/pipeline/source-assets/)
+* Non-asset source files
+   * Non-asset source files are files that may or may not be in a scan directory, that are loaded and referenced when processing a source asset.
+   * The difference between a source asset and a non-asset source file is either:
+      * The file is not in a scan directory,
+      * The file does not match a file pattern for any asset builder descriptor, or
+      * The file does not generate a job when create jobs is called for the file for matching asset builders
+* Product assets
+   * Product assets are the run-time ready output of an asset processing job.

```

Comment:
Minor, we're trying to use runtime consistently rather than run-time.

```suggestion
   * Product assets are the runtime ready output of an asset processing job.
```